libsolidity/analysis/ContractLevelChecker.cpp

```diff
@@ -621,58 +621,38 @@ void ContractLevelChecker::checkBaseABICompatibility(ContractDefinition const& _
 
 void ContractLevelChecker::checkAmbiguousOverrides(ContractDefinition const& _contract) const
 {
-	vector<FunctionDefinition const*> contractFuncs = _contract.definedFunctions();
-
-	auto const resolvedBases = resolveDirectBaseContracts(_contract);
-
-	FunctionMultiSet inheritedFuncs = inheritedFunctions(&_contract);;
+	// Fetch non overridden base functions and sort them by signature.
+	FunctionMultiSet nonOverriddenBaseFunctionSet = convertContainer<decltype(nonOverriddenBaseFunctionSet)>(
```

Comment:
Isn't mentioning `FunctionMultiSet` explicitly better here?

Reply:
I also wonder whether we should rename ``FunctionMultiSet`` to ``SignatureSortedFunctionSet``...